keepassxc-browser/content/fill.js

```diff
@@ -250,14 +246,20 @@ kpxcFill.fillInCredentials = async function(combination, predefinedUsername, uui
     }
 
     // Fill password
-    if (combination.password) {
+    if (combination.password && combination.password.nodeName === 'INPUT') {
         // Show a notification if password length exceeds the length defined in input
         if (combination.password.maxLength
             && combination.password.maxLength > 0
             && selectedCredentials.password.length > combination.password.maxLength) {
             kpxcUI.createNotification('error', tr('errorMessagePaswordLengthExceeded'));
         }
 
+        // Prevent filling password to plain text input field
+        if (passOnly && !passwordFillIsAllowed(combination.password)) {
+            kpxcUI.createNotification('warning', tr('fieldsPasswordFillNotAccepted'));
```

Comment:
The above check is an error and allowed to proceed. This one is a warning and stopped. Those need to be swapped in this case (the error and warning attributes).